ui/app/src/components/wallet/KeplrWalletPanel.vue

```diff
@@ -1,50 +1,35 @@
 <script lang="ts">
 import { computed, defineComponent } from "vue";
 import { useCore } from "@/hooks/useCore";
-import BalanceTable from "./BalanceTable.vue";
 import SifButton from "@/components/shared/SifButton.vue";
 import Icon from "@/components/shared/Icon.vue";
 
-function useKeplrWallet() {
-  const { store, actions } = useCore();
-  async function handleDisconnectClicked() {
-    await actions.wallet.disconnectWallet();
```

Comment:
Should we remove the usecase for this if we are not using it?

Reply:
yeah, less code to maintain is better